channels/layers.py

```diff
@@ -225,13 +225,14 @@ async def send(self, channel, message):
         # name in message
         assert "__asgi_channel__" not in message
 
-        queue = self.channels.setdefault(channel, asyncio.Queue())
-        # Are we full
-        if queue.qsize() >= self.capacity:
-            raise ChannelFull(channel)
-
+        queue = self.channels.setdefault(
+            channel, asyncio.Queue(maxsize=self.get_capacity(channel))
+        )
```

Comment:
This changes the semantics around queue size right? What happens if the capacity is changed by the user while it is being used? I think that the new code would not honor that capacity change right?

Reply:
good point. I am not sure about it. Is it really wished, that the capacity can change during runtime? Also it just affects not newly created channels.

Reply:
Hm I suppose not. @carltongibson  do you have any thoughts?

Reply:
I've thought about it and I can't think of any realistic scenario where this might cause problems, so I think the change is acceptable and we can document the change in the release notes